src/Microsoft.VisualStudio.Editors/Common/Utils.vb

```diff
@@ -1566,6 +1566,56 @@ Namespace Microsoft.VisualStudio.Editors.Common
         ''' <remarks></remarks>
         Friend Function IsTargetingDotNetFramework45OrAbove(hierarchy As IVsHierarchy) As Boolean
 
+            Dim frameworkName As FrameworkName = Nothing
+            If TryGetTargetFrameworkMoniker(hierarchy, frameworkName) Then
+                If String.Compare(frameworkName.Identifier, ".NETFramework", StringComparison.OrdinalIgnoreCase) = 0 AndAlso
+                (frameworkName.Version.Major > 4 OrElse (frameworkName.Version.Major = 4 AndAlso frameworkName.Version.Minor >= 5)) Then
+                    ' Targeting .NET framework 4.5 or higher
+                    Return True
+                ElseIf IsTargetingDotNetCore(frameworkName.Identifier) Then
```

Comment:
I'm rather concerned by this. Is this a public function that other people consume, or just internal? Because if it's just internal, we shouldn't return true for this case: it's *not* targeting .netframework 4.5 or above, it's targeting .netcore 1.0. And even if it is public, we should fix the thing calling this, not return true when the condition is actually false.